### ISSUE TITLE
Websocket error with message "KeyError: 'supported_brands'" upon adding integration 

### ISSUE THREAD

**Issue by nottveitvegen210** [home-assistant/frontend#13543]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When clicking on ADD INTEGRATION, the modal dialog opens, but stays on the spinner.

Error message in browser console is:
```
step-flow-progress.ts:11 
 Uncaught (in promise) 
{code: 'unknown_error', message: 'Unknown error'}
value	@	step-flow-progress.ts:11
await in value (async)		
c	@	ha-svg-icon.ts:5
await in c (async)		
(anonymous)	@	ha-svg-icon.ts:5
i	@	typography.js:181
o	@	94eb1ba9.js:1
l	@	ha-fab.ts:28
value	@	ha-config-integrations.ts:533
handleEvent	@	styles.ts:370
```

Additionally, in the Home Assistant log, I find the following entry:

```
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/commands.py:724
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 21:06:48 (1 occurrences)
Last logged: 21:06:48

[140422837459824] Error handling message: Unknown error (unknown_error)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 27, in _handle_async_response
    await func(hass, connection, msg)
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 724, in handle_supported_brands
    data[int_or_exc.domain] = int_or_exc.manifest["supported_brands"]
KeyError: 'supported_brands'
```

### Describe the behavior you expected

Getting list of integrations that can be added to my Home Assistant installation.

### Steps to reproduce the issue

1. Clicking on ADD INTEGRATION.

### What version of Home Assistant Core has the issue?

2022.08.5

### What was the last working version of Home Assistant Core?

2022.07

### In which browser are you experiencing the issue with?

Edge / Chrome

### Which operating system are you using to run this browser?

Windows 11

### Javascript errors shown in your browser console/inspector

```txt
See problem description.
```

### Additional information

In the forums (see https://community.home-assistant.io/t/could-not-add-new-integrations/447935), it is discussed that the following commit https://github.com/home-assistant/core/commit/61cc9f5288f378529daa122062b6120d50d1dffd and PR https://github.com/home-assistant/core/pull/75176 might be related to the issue.

Below also the system information of my Home Assistant installation.

## System Information

version | core-2022.8.7
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.10.5
os_name | Linux
os_version | 5.15.60
arch | x86_64
timezone | Europe/Oslo
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4617
Installed Version | 1.27.1
Stage | running
Available Repositories | 1176
Downloaded Repositories | 19

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | July 11, 2023 at 02:00
relayer_connected | true
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | true
remote_server | eu-west-2-0.ui.nabu.casa
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Easee EV Charger</summary>

component_version | 0.9.44
-- | --
reach_easee_cloud | ok
connected2stream | true

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 8.5
-- | --
update_channel | stable
supervisor_version | supervisor-2022.08.5
agent_version | 1.2.1
docker_version | 20.10.14
disk_total | 117.4 GB
disk_used | 11.0 GB
healthy | true
supported | true
board | ova
supervisor_api | ok
version_api | ok
installed_addons | SSH & Web Terminal (12.0.2), Z-Wave JS (0.1.68), Studio Code Server (5.3.0), Samba share (10.0.0), Home Assistant Google Drive Backup (0.108.4), Mosquitto broker (6.1.3)

</details>

<details><summary>Dashboards</summary>

dashboards | 9
-- | --
resources | 7
views | 20
mode | storage

</details>

<details><summary>Nibe Uplink Heat Pumps</summary>

api_endpoint_reachable | ok
-- | --

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | August 27, 2022 at 02:12
-- | --
current_recorder_run | August 31, 2022 at 21:03
estimated_db_size | 458.71 MiB
database_engine | sqlite
database_version | 3.38.5

</details>

**Comment by rogerbille** [comment]
Same problem, had to go back to 2022.7.7.

**Comment by nottveitvegen210** [comment]
I just updated to 2022.9.2 and ... I am again getting the list of integrations in the "Set up a new integration" dialog. Seems that for me, the problem is solved. Maybe other can comment if it is solved for them, too?

**Comment by emontnemery** [comment]
This was indeed fixed in 2022.9.2 by https://github.com/home-assistant/core/pull/78090
